prime-router/metadata/fhir_mapping/hl7/datatype/Identifier.yml

```diff
@@ -0,0 +1,24 @@
+type:
+   valueOf: datatype/CodeableConcept
+   condition: $code NOT_NULL || $alternateCode NOT_NULL || $type NOT_NULL
+   expressionType: resource
+   specs: CX.5 | $type
```

Comment:
Could CX.5 be not null and generate some data?

Reply:
Changed it to use a combined var to include CX.5 in the checking